generators/client/templates/vue/webpack/webpack.dev.js.ejs

```diff
@@ -24,7 +24,7 @@ module.exports = webpackMerge(baseWebpackConfig, {
     main: './<%= MAIN_SRC_DIR %>app/main'
   },
   output: {
-    path: jhiUtils.root('<%= DIST_DIR %>'),
+    path: path.resolve('<%= DIST_DIR %>'),
```

Comment:
```suggestion
    path: path.resolve(__dirname, '..', '<%= DIST_DIR %>'),
```
Not sure but without this npm start could fail if you are not at the project root.

Reply:
did that